contracts/Staking.sol

```diff
@@ -19,6 +19,7 @@ contract Staking is Autopetrified, ERCStaking, ERCStakingHistory, IStakingLockin
     string private constant ERROR_TOKEN_NOT_CONTRACT = "STAKING_TOKEN_NOT_CONTRACT";
     string private constant ERROR_AMOUNT_ZERO = "STAKING_AMOUNT_ZERO";
     string private constant ERROR_TOKEN_TRANSFER = "STAKING_TOKEN_TRANSFER";
+    string private constant ERROR_TOKEN_DEPOSIT = "STAKING_TOKEN_DEPOSIT";
```

Comment:
I'd use `STAKING_TOKEN_DEPOSIT_FAIL`

Reply:
Ok, I’ll open another PR for it and change the one for transfer accordingly.